docs/conf.py

```diff
@@ -68,6 +68,8 @@
     ("py:class", "'id'"),
     ("py:class", "typing_extensions.Literal"),
     ("py:func", "int"),
+    ("py:class", "pandablocks.asyncio.FlushMode"),
```

Comment:
Then this ignore shouldn't be needed

Reply:
Without it docs won't build with error:
```
pandablocks/asyncio.py:docstring of pandablocks.asyncio.AsyncioClient.data:1: WARNING: py:class reference target not found: pandablocks.asyncio.FlushMode 
```

Likewise for `asyncio.locks.Event`.